pnpm-lock.yaml

```diff
@@ -2773,7 +2776,7 @@ packages:
       hardhat: ^2.0.0
     dependencies:
       ethers: 5.7.2
-      hardhat: 2.17.2(ts-node@10.9.1)(typescript@5.2.2)
```

Comment:
I wonder why this downgraded

Reply:
I have no idea but I don't think we should mess with it since CI works and this is a generated file (these changes are from running `pnpm install --save-dev node-fetch`)